superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -241,6 +253,8 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     }
     if (show) {
       setTabKey(DEFAULT_TAB_KEY);
+      getAvailableDbs();
+      setSelectedDb('postgresql');
```

Comment:
this is hardcoded for now, and doesn't do anything unless useSqlAlchemyForm is true. Eventually the db will be chosen by the user on a different step.